plugins/woocommerce/client/admin/client/settings-payments/components/modals/woo-payments-reset-account-modal.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Button, Modal } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './modals.scss';
+import { getWooPaymentsResetAccountLink } from '~/settings-payments/utils';
+
+interface WooPaymentsResetAccountModalProps {
+	isOpen: boolean;
+	onClose: () => void;
+}
+
+export const WooPaymentsResetAccountModal = ( {
+	isOpen,
+	onClose,
+}: WooPaymentsResetAccountModalProps ) => {
+	const [ isResettingAccount, setIsResettingAccount ] = useState( false );
+
+	const handleResetAccount = () => {
+		setIsResettingAccount( true );
+
+		window.location.href = getWooPaymentsResetAccountLink();
+	};
+
+	return (
+		<>
+			{ isOpen && (
+				<Modal
+					title={ __( 'Reset your test account', 'woocommerce' ) }
+					className="woocommerce-woopayments-modal"
+					isDismissible={ true }
+					onRequestClose={ onClose }
+				>
+					<div className="woocommerce-woopayments-modal__content">
+						<div className="woocommerce-woopayments-modal__content__item">
+							<div>
+								<span>
```

Comment:
Can we use a `<p>` tag here instead?

Reply:
yeah, good idea!

Reply:
done in https://github.com/woocommerce/woocommerce/pull/53590/commits/06ea5fe011abd6832bd303c07bde94e820efbde7!